torch/autograd/profiler.py

```diff
@@ -802,15 +827,21 @@ def self_cpu_time_total(self):
 
     @property
     def cuda_time_total(self):
+        if self.device_id >= 0:
```

Comment:
why is device_id here? is it the kineto different format?

Reply:
it's a hacky workaround - will remove shortly